### PR TITLE
Add wrapper for internal use of interpolation functions

### PR DESCRIPTION
**Pull request by LasNikas**
We should then resume discussion in #424 and if we want to have `v_ode` and `u_ode` in `custom_quantities`.
Also, it might make sense to have `dv_ode` with `get_du` in `custom_quantities`

### CODE REVIEW

**Comment by svchb** [comment]
Conflict with #462

**Comment by LasNikas** [comment]
Ah didn't notice. But why should we create a named tuple which is then unpacked again in the method?

**Comment by svchb** [comment]
Yes I wanted to mention that this conflicts. I also thought about this but I wanted to keep the changes minimal in #462.

**Review by svchb — APPROVED** [review]

File: src/general/interpolation.jl [hunk 1]
```diff
@@ -51,14 +51,33 @@ results = interpolate_plane_2d([0.0, 0.0], [1.0, 1.0], 0.2, semi, ref_system, so
...
 """
-function interpolate_plane_2d(min_corner, max_corner, resolution, semi, ref_system, sol;
+function interpolate_plane_2d(min_corner, max_corner, resolution, semi, ref_system,
+                              sol::ODESolution;
                               smoothing_length=ref_system.smoothing_length,
                               cut_off_bnd=true, clip_negative_pressure=false)
     # Filter out particles without neighbors
     filter_no_neighbors = true
+    v_ode = sol.u[end].x[1]
+    u_ode = sol.u[end].x[2]
```
Comment (efaulhaber) [inline src/general/interpolation.jl:61]:
```suggestion
    v_ode, u_ode = sol.u[end].x
```

**Review by efaulhaber — APPROVED** [review]

File: src/general/interpolation.jl (hunk 1 above)
Comment (efaulhaber) [inline src/general/interpolation.jl:61]:
```suggestion
    v_ode, u_ode = sol.u[end].x
```

**Review by efaulhaber — CHANGES_REQUESTED** [review]

**Review by svchb — APPROVED** [review]

**Review by svchb — APPROVED** [review]

**Pull request merged**